sdk/eventhub/event-hubs/test/internal/sender.spec.ts

```diff
@@ -725,7 +725,7 @@ describe("EventHub Sender", function(): void {
         ]
       };
 
```

Comment:
I can't wait to move these test methods to `@azure/test-utils` after this upgrade